src/css/components/grid.scss

```diff
@@ -1,7 +1,12 @@
 .grid {
-  @include outer-container;
```

Comment:
outer-container contained a max-width property that was not needed and causing issue in the PL.